api/test/singleton/singleton_test.cc

```diff
@@ -316,6 +342,12 @@ void cleanup_otel()
   trace_api::Provider::SetTracerProvider(provider);
 }
 
+#ifdef _WIN32
+#  define RUN_FAILING_WINDOWS_TEST 0
```

Comment:
Add TODO item to enable this once it is fixed?

Reply:
Fixed.